docs/api-reference.md

```
@@ -981,18 +1015,58 @@ interface ValidationDetail {
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)